### PR TITLE
update rapidsai/ci to rapidsai/ci-conda

### PR DESCRIPTION
**Pull request by AyodeAwe**
PR updates `rapidsai/ci` references to `rapidsai/ci-conda`.

### CODE REVIEW

**Comment by ajschmidt8** [comment]
FWIW, this PR is effectively a no-op. We currently publish the same images to both the `rapidsai/ci` and `rapidsai/ci-conda` tags.

Once we've removed all the references to `rapidsai/ci`, we will remove them from DockerHub.

**Review by mdemoret-nv — APPROVED** [review]

**Review by dagardner-nv — APPROVED** [review]

File: .github/workflows/pull_request.yml [hunk 1]
```diff
@@ -39,7 +39,7 @@ jobs:
     runs-on: linux-amd64-cpu4
     timeout-minutes: 60
     container:
-      image: rapidsai/ci:cuda11.5.1-ubuntu20.04-py3.8
+      image: rapidsai/ci-conda:cuda11.5.1-ubuntu20.04-py3.8
```
Comment (ajschmidt8) [inline .github/workflows/pull_request.yml:42]:
```suggestion
      image: rapidsai/ci-conda:latest
```

It looks like this particular image tag is no longer being published.

We should switch to using the `latest` alias here if this job doesn't depend on any particular CUDA/OS/Python versions.

Thoughts, @dagardner-nv?

**Comment by ajschmidt8** [comment]
merging since this has been approved

**Pull request merged**